src/libraries/System.Xml.XmlSerializer/ref/System.Xml.XmlSerializer.cs

```diff
@@ -585,6 +585,7 @@ protected partial class Fixup
     public abstract partial class XmlSerializationWriter : System.Xml.Serialization.XmlSerializationGeneratedCode
     {
         protected XmlSerializationWriter() { }
+        protected readonly char[] primitivesBuffer;
```

Comment:
This feels a bit weird. Are there other places in ref assemblies where protected fields are exposed?